src/BlazorWebView/src/Maui/iOS/BlazorWebViewHandler.iOS.cs

```diff
@@ -95,10 +95,12 @@ protected override WKWebView CreatePlatformView()
 				AutosizesSubviews = true
 			};
 
-#if MACCATALYST13_3_OR_GREATER || IOS16_4_OR_GREATER
-			// Enable Developer Extras for Catalyst/iOS builds for 16.4+
-			webview.SetValueForKey(NSObject.FromObject(DeveloperTools.Enabled), new NSString("inspectable"));
-#endif
+			if (OperatingSystem.IsIOSVersionAtLeast(13) || OperatingSystem.IsMacCatalystVersionAtLeast(13, 1)) 
```

Comment:
This matches the old versions:

```suggestion
			if (OperatingSystem.IsIOSVersionAtLeast(16, 4) || OperatingSystem.IsMacCatalystVersionAtLeast(13, 3)) 
```

Reply:
Yep, I downloaded the 15.5 Simulator to verify and it does throw if run on it. I was about to change it before you got to it. Thanks again!